src/Microsoft.VisualStudio.Services.Agent/Util/VarUtil.cs

```diff
@@ -1,12 +1,48 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 
 namespace Microsoft.VisualStudio.Services.Agent.Util
 {
-    public static class VarUtil
+    [ServiceLocator(Default = typeof(VarUtil))]
+    public interface IVarUtil : IAgentService
     {
+        void PrependPath(string directory);
+        void SetEnvironmentVariable(string name, string value);
+    }
+
+    public sealed class VarUtil : AgentService, IVarUtil
+    {
+        public void PrependPath(string directory)
```

Comment:
I put those behind an interface for testability. Those methods modify the environment.

The two existing static methods are probably better not to abstract. Due to the nature of those methods, my gut is it actually makes tests more cumbersome to write if they were behind an interface, rather than just use the actual methods.